motorlib/simResult.py

```diff
@@ -143,6 +150,11 @@ def getAveragePressure(self):
     def getMaxPressure(self):
         """Returns the highest chamber pressure that was observed during the motor's burn."""
         return self.channels['pressure'].getMax()
+        
+    def getMinExitPressure(self):
+        """Returns the lowest exit pressure that was observed during the motor's burn, ignoring startup and shutdown transients"""
+        exit_pressures = self.channels['exitPressure'].getData()
+        return min(exit_pressures[5:-5])
```

Comment:
I get wary whenever I see hardcoded magic numbers like this. I wonder if we should make this condition based on some percentage, like:
Motor had N% of total burn time where the exit pressure was under the threshold 
Where N is a config value.